vignettes/theme.Rmd

```diff
@@ -0,0 +1,321 @@
+---
+title: "Theming flexdashboard"
+resource_files:
+  - snippets
+---
+
+```{r setup, include=FALSE}
+library(quillt)
+```
+
+## Using Bootstrap
+
+The `flex_dashboard` output format gives you access to the [Bootstrap CSS framework](https://getbootstrap.com/). Bootstrap is a library of reusable chunks of HTML, CSS, and JavaScript code that you can call on to modify your flexdashboard without writing that code yourself. Originally [created by and for Twitter](https://en.wikipedia.org/wiki/Bootstrap_(front-end_framework)), it is a free and open source project; you can view the repository on [GitHub](https://github.com/twbs/bootstrap). 
+
+The `flex_dashboard` output format includes a `theme` parameter for customizing the overall appearance of the dashboard. This parameter has special awareness of the [`{bslib}` package](https://rstudio.github.io/bslib/), which provides tools for using the [Bootstrap CSS framework](https://getbootstrap.com/) directly from R. For now, you'll need to install the development version of the package to use it:
+
+``` {.r}
+# install.packages("remotes")
+remotes::install_github("rstudio/bslib")
+```
```

Comment:
Just to be sure here: dev **bslib** is required for this vignettes ? 
Dev **flexdashboard** also right ?

I am starting thinking about what to write for **rmarkdown** so I am curious. Thanks.

Reply:
Strictly speaking it's not necessary, but there are some important fixes/additions

Reply:
Perfect. Thanks ! 

I think will update the requirement in **rmarkdown** when you version hit CRAN then. thanks!